test/unit/src/audio/FftUnit.cpp

```diff
@@ -3,7 +3,7 @@
 // FIXME: OOURA roundtrip FFT seems to be broken on windows for sizeFft = 4 (https://github.com/cinder/Cinder/issues/1263)
 #if ! defined( CINDER_MSW )
 
-#include "catch.hpp"
+#include "../catch.hpp"

```

Comment:
Shouldn't the header search path for the tests be consistent so we don't need to describe relative paths in the test files themselves?